docs/quickstarts/ansible-create-first-playbook/metadata.yml

```diff
@@ -0,0 +1,5 @@
+kind: Quickstarts # kind must always be "Quickstarts"
+name: ansible-create-first-playbook
+tags: # If you want to use more granular filtering add tags to the quickstart
+  - kind: bundle
+    value: Ansible
```

Comment:
This might need to be lowercase. I'm not sure but you can test it out in the [preview tool](https://quickstarts-content-preview.surge.sh/)